### ISSUE TITLE
anvi-gen-samples-information-db sorts layer names alphabetically

### ISSUE THREAD

**Issue by ozcan** [merenlab/anvio#518]
@tdelmont thinks this is is confusing, and it should follow the order the in the file.

these lines are related with this issue:
[https://github.com/merenlab/anvio/blob/master/anvio/dbops.py#L2086-L2088
]()

**Comment by ozcan** [comment]
@meren what do you think? should we remove sorted() from here?

**Comment by meren** [comment]
I agree. But we need more flexibility there. Let's remove it for now, and play with it a little more using a samples db with samples information and samples order (both newick and basic form) to see how it plays out.
